Sources/JSEnvironment.swift

```diff
@@ -0,0 +1,332 @@
+//
```

Comment:
To begin with, yeah, after that we'll see about moving it to a submodule. I can imagine some people using StencilSwiftKit on platforms where there's no JavascriptCore support.